src/pybind/mgr/dashboard/frontend/src/app/shared/datatable/table/table.component.ts

```diff
@@ -454,8 +454,12 @@ export class TableComponent implements AfterContentChecked, OnInit, OnChanges, O
     this.onSelect();
```

Comment:
`ERROR in src/app/shared/datatable/table/table.component.ts(454,5): error TS2554: Expected 1 arguments, but got 0.`

```suggestion
    this.onSelect(this.selection);
```